src/app/progress/PursuitItem.m.scss

```diff
@@ -29,6 +29,7 @@
   background-position: center;
   background-repeat: no-repeat;
   background-size: contain;
+  filter: saturate(0) brightness(0.75) invert(var(--theme-header-characters-icon));
```

Comment:
Agree, this was failed test to see if the white icons could consistently be inverted/colourised as grey for the light theme. 

But since there isn't really any way to delineate between the monotone icons vs image-based icons the cleanest approach here feels like it'd be adding a variable to set a background colour for the icons on any light themes